src/demos/multi_file_viewer_demo.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <div class="multi-file-viewer-demo">
+
+    <div>
+      <p @click="open_file(1)"> Add A </p>
+      <p @click="open_file(2)"> Add B </p>
+      <p @click="open_file(3)"> Add C </p>
+      <p @click="open_file(4)"> Add D </p>
+      <p @click="open_file(5)"> Add E </p>
+      <div class="mvf-container">
+        <multi-file-viewer
+          ref="multi_file"
+          :height_of_view_file="height_of_view_file">
+        </multi-file-viewer>
+      </div>
+    </div>
+
+    <div class="faux-incorrect-row"> </div>
+    <div class="faux-correct-row"> </div>
+    <div class="faux-unavailable-row"> </div>
+
+
+    <div>
+      <p class="letters" @click="open('A')"> Add A </p>
+      <p class="letters" @click="open('B')"> Add B </p>
+      <p class="letters" @click="open('C')"> Add C </p>
+      <div class="mvf-container">
+        <multi-file-viewer
+          ref="multi_file2">
+        </multi-file-viewer>
+      </div>
+    </div>
+
+  </div>
+</template>
+
+<script lang="ts">
+
+  import MultiFileViewer from '@/components/multi_file_viewer.vue';
+  import { Component, Prop, Vue } from 'vue-property-decorator';
+
+  @Component({
+    components: { MultiFileViewer }
+  })
+  export default class MultiFileViewerDemo extends Vue {
+
+    open_file(file_number: number) {
+      let mfv = <MultiFileViewer> this.$refs.multi_file;
+      if (file_number === 1) {
+        mfv.add_to_viewing(this.file1, this.file_1_content);
+      }
+      else if (file_number === 2) {
+        mfv.add_to_viewing(this.file2, this.file_2_content);
+      }
+      else if (file_number === 3) {
+        mfv.add_to_viewing(this.file3, this.file_3_content);
+      }
+      else if (file_number === 4) {
+        mfv.add_to_viewing(this.file4, this.file_4_content);
+      }
+      else {
+        mfv.add_to_viewing(this.file5, this.file_5_content);
+      }
+    }
+
+    open(file: string) {
+      let mfv = <MultiFileViewer> this.$refs.multi_file2;
+      if (file === 'A') {
+        mfv.add_to_viewing(this.file_a, this.file_a_content);
+      }
+      else if (file === 'B') {
+        mfv.add_to_viewing(this.file_b, this.file_b_content);
+      }
+      else {
+        mfv.add_to_viewing(this.file_c, this.file_c_content);
+      }
+    }
+
+    file_a = 'A';
+    file_b = 'B';
+    file_c = 'C';
+
+    file_a_content = "Aurora\nAurora";
+    file_b_content = "Belle";
+    file_c_content = "Cinderella\nCinderella\nCinderella";
+
+
+
+    file1 = "Apple";
```

Comment:
I like vertical whitespace too, but these variables should probably not have a blank line between all of them